roles/router/templates/systemd/20-aether-core.network

```diff
@@ -10,4 +10,4 @@ Address={{ core.upf.core_subnet }}
 
 [Route]
 Gateway={{ core.upf.default_upf.ip.core }}
-Destination={{ core.upf.default_upf.ue_ip_pool }}
+Destination={{ core.upf.default_upf.ue_ip_pool }}
```

Comment:
You need to add an empty line to the files that you modified and have the red symbol